src/Tools/Source/CompilerGeneratorTools/Source/BoundTreeGenerator/BoundNodeClassWriter.cs

```diff
@@ -1530,22 +1530,22 @@ private void WriteNullabilityRewriter()
                             Unbrace();
 
                             void writeNullabilityCheck(bool inverted) =>
-                                WriteLine($"if ({(inverted ? "!" : "")}{updatedNullabilities}.TryGetValue(node, out TypeSymbolWithAnnotations type))");
+                                WriteLine($"if ({(inverted ? "!" : "")}{updatedNullabilities}.TryGetValue(node, out (NullabilityInfo Info, TypeSymbol Type) infoAndType))");
```

Comment:
>(NullabilityInfo Info, TypeSymbol Type) infoAndType) [](start = 116, length = 52)

nit: ordering? "type and info" seems more natural

Reply:
I did it in this order because this felt more natural to me, and is in line with how I would want it to be printed. Types can be long, so having the bounded-length items first makes sense to me.

---
In reply to: [264939378](https://github.com/dotnet/roslyn/pull/34017#discussion_r264939378) [](ancestors = 264939378)